testcases/MachineConfig.py

```diff
@@ -483,7 +496,19 @@ def __init__(self, cv_HMC=None, system_name=None,
         self.lmb_size = lmb
         self.num_hugepages = hugepages
         self.setup = 0
-        self.cec_dict = {'lmb_cec': None, 'hugepages': None}
+        self.iocapacity = iocapacity
+        self.cec_dict = {'lmb_cec': None, 'hugepages': None, 'iocapacity': None}
+        self.config = OpTestConfiguration.conf
+        self.bmc_type=bmc_type
+        if self.bmc_type == "EBMC_PHYP":
+            self.BMCHostMgmt = EBMCHostManagement(conf=self.config,
+                                                  ip=self.config.args.bmc_ip,
+                                                  username=self.config.args.bmc_username,
+                                                  password=self.config.args.bmc_password)
+        elif self.bmc_type == "FSP_PHYP":
+            self.cv_ASM = OpTestASM(self.args.bmc_ip,
+                                    self.args.bmc_username,
+                                    self.args.bmc_password)
```

Comment:
I do not feel a need to instantiate the object explicitly, as when the cv_System  objects is intanstiated.. you get cv_FSP and you can use the same object to call self.cv_FSP.configure_enlarged_io() which will internal created the cv_ASM object.. try this

Reply:
also similarly we can use self.cv_BMC.io_enlarge_capacity() can be directly used

Reply:
Yes, Trying to allign the code to work with both bmc and FSP